src/discof/replay/fd_replay_tile.c

```diff
@@ -327,8 +327,9 @@ struct fd_replay_tile {
   fd_block_id_map_t * block_id_map;
 
   /* Capture-related configs */
-  fd_capture_ctx_t * capture_ctx;
-  FILE *             capture_file;
+  fd_capture_ctx_t *     capture_ctx;
+  FILE *                 capture_file;
```

Comment:
Why does the replay tile still hold on to a file handle if the writing logic has been moved to the solcap tile?